src/dev/code_coverage/ingest_coverage/teams_scripted_field.painless

```diff
@@ -0,0 +1,87 @@
+String path = doc['coveredFilePath.keyword'].value;
+if (path.contains('/apm'))  return "APM";
+ else if (path.contains('/canvas')) return "Canvas";
+ else if (path.contains('/maps')) return "Maps";
+ else if (path.contains('map_')) return "Maps";
+ else if (path.contains('/ml')) return "ML";
```

Comment:
This will match a lot of files outside the ML app (and not written / maintained by the ML team), a few examples:
- `x-pack/legacy/plugins/siem/public/components/ml/anomaly/translations.ts` -> SIEM
- `x-pack/legacy/plugins/uptime/public/components/monitor/ml/ml_flyout.tsx` -> Observability
- `x-pack/legacy/plugins/monitoring/public/views/elasticsearch/ml_jobs/index.js` -> Stack Monitoring

The same goes for other file path parts as well, e.g. `x-pack/plugins/security/public/management/roles/edit_role/privileges/kibana/transform_error_section/index.ts` is not part of the transform plugin and is not maintained by the ML team.

Reply:
For the ML team, it would probably be sufficient to check for
- `plugins/ml`
- `plugins/transform`
But not sure if this approach is applicable to all parts of the code.

Reply:
Ok, I'll ping @LeeDr , that's his baby :)

Reply:
@pheyos I can tell you that we are indeed considering moving this logic into NodeJS, on the ingestion side of the house

Reply:
@pheyos I'm making this change live.
On the most recent run with the current scripted field and filtering on team:ML I get 2,035 hits (out of unfiltered 17,877 hits)
With this change;
 else if (path.contains('/plugins/ml')) return "ML";
 else if (path.contains('plugins/transform')) return "ML";

Now 1,911 hits

I'll paste the updated scripted field into this PR.

But also note that the scripted field is very large and is causing problems in a few areas in Kibana so we (in another PR) be removing it and applying the team labels either in ingestion, or with a ingest pipeline in Elasticsearch.  And a possible update_by_query as needed.

Reply:
@pheyos if you wouldn't mind checking again.  Should be better.

Reply:
@LeeDr thanks for adjusting!
From the ML point of view the current assignment looks good.